apps/prairielearn/src/sync/syncFromDisk.ts

```diff
@@ -106,6 +126,7 @@ export async function syncDiskToSqlWithLock(
       syncQuestions.sync(courseId, courseData),
     );
 
+    await timed('Sync sharing sets', () => syncSharingSets.sync(courseId, courseData, questionIds));
```

Comment:
Please match all the existing code:

```suggestion
    await timed('Synced sharing sets', () => syncSharingSets.sync(courseId, courseData, questionIds));
```